#from logging import exception
#from traceback import print_exception
import matplotlib.pyplot as plt

#try:
#years = [1,2,3,4,5]
#values = [4,5,7,9,1]
#ruta = "C:\Resilient\imagen.jpg"

#plot = plt.bar(years,values)
#plt.title("Grafico")
#plt.xlabel("year")
#plt.ylabel("values")

#plt.savefig(ruta)

#except exception as e:
 #   print_exception()
  #  raise e 

